docker/ubuntu/baseimage/install_luarocks.sh

```diff
@@ -1,30 +1,11 @@
 #!/usr/bin/env bash
 
-HOME=/home/ko
+apt-get install --no-install-recommends luarocks
```

Comment:
@yparitcher This cannot have ever worked, can it?

Reply:
No, it cannot. 

If we want to use distro luarocks, it should be done in https://github.com/koreader/virdevenv/blob/11b62a5722b3adc1d98ce8484b89968b136db4c5/docker/ubuntu/baseimage/bootstrap.sh really

But you could do it here with sudo in theory, see https://github.com/koreader/virdevenv/blob/11b62a5722b3adc1d98ce8484b89968b136db4c5/docker/ubuntu/baseimage/Dockerfile#L17

Keep in mind we purposefully remove the apt cache lines earlier https://github.com/koreader/virdevenv/blob/11b62a5722b3adc1d98ce8484b89968b136db4c5/docker/ubuntu/baseimage/Dockerfile#L6